docs/user-guide/tree_structures.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p>I'd move this and the grid construction above the "BallTree" section, under the import. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Capitalize UGRID </p>